client/src/components/audit-row/AuditRow.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+
+import { TableCell, TableRow, Typography } from '@mui/material';
+
+import { Audit } from '../../types';
+import resolveAuditMessage from './resolver';
+
+export interface AuditRowProps {
+  audit: Audit;
+}
+
+const AuditRow: React.FC<AuditRowProps> = ({ audit }) => {
+  const dateTime = new Date(audit.createdAt).toLocaleString();
+
+  return (
+    <TableRow hover sx={{ cursor: 'pointer' }}>
+      <TableCell style={{ width: '180px' }}>
+        <Typography variant="caption">{dateTime}</Typography>
+      </TableCell>
+      <TableCell>
+        <Link to={`/staff/${audit.staffId}`}>{audit.createdBy.email}</Link>
```

Comment:
Wouldn't the name be more useful?

Reply:
Could have staff with the same first name and last name, email is unique

Reply:
i guess that is a reason